docroot/profiles/custom/cgov_site/modules/custom/cgov_core/src/Plugin/metatag/Group/CgdpAdvanced.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Drupal\cgov_core\Plugin\metatag\Group;
+
+use Drupal\metatag\Plugin\metatag\Group\GroupBase;
+
+/**
+ * Provides a plugin for the 'Cgdp Additional Tags' meta tag group.
+ *
+ * @MetatagGroup(
+ *   id = "cgdpAdvanced_advanced",
+ *   label = @Translation("Cgdp Additional Tags"),
+ *   description = @Translation("These tags are not part of the Metadata Element Set but may be useful for certain scenarios."),
+ *   weight = 4,
+ * )
+ */
+class CgdpAdvanced extends GroupBase {
+  // Nothing here yet. Just a placeholder class for a plugin.
```

Comment:
If this is never going to get code can you just make the comment, "Just a placeholder class for a plugin."